less/about/bookmarks.less

```diff
@@ -1,4 +1,8 @@
-@import "./siteDetails.less";
```

Comment:
Moved to https://github.com/brave/browser-laptop/pull/10881/files#diff-c74077737ab01e5191e480529919c489R24